### PR TITLE
Removed distracting note from tutorial 4.

### PR DESCRIPTION
**Pull request by carltongibson**
The note on a possible race condition is inappropriate in this tutorial setting. To quote Diátaxis:

> Your job is to guide the learner to a successful conclusion. There
> may be many interesting diversions along the way … - ignore them.

### CODE REVIEW

**Comment by carltongibson** [comment]
* https://diataxis.fr/tutorials/#ignore-options-and-alternatives
* https://mastodon.social/@webology/111964426381142529

**Comment by carltongibson** [comment]
@ryanhiebert suggested just fixing the example, which is likely the best route here (and matches Tim's comment on the ticket, many moons ago)

**Comment by ryanhiebert** [comment]
At least one commenter on the ticket felt that the F expressions were too complicated, but I do think that Diátaxis has helped clarify that tutorials don't need to explain everything. F expressions are an interesting feature that they may not fully understand, but that's OK. We still want the newcomer to see them, even if they don't fully understand it.

I think that a very brief note in regular prose that this new `F` expression avoids a race condition, and linking to more substantive documentation for those that wish to follow that rabbit trail, provides a really good balance for a tutorial.

**Comment by carltongibson** [comment]
I think I agree @ryanhiebert. It's my bedtime. Fancy putting a PR to my branch?

File: docs/intro/tutorial04.txt [hunk 1]
```diff
@@ -97,7 +97,7 @@ create a real version. Add the following to ``polls/views.py``:
                 },
             )
         else:
-            selected_choice.votes += 1
+            selected_choice.votes = F("votes") + 1
```
Comment (felixxm) [inline docs/intro/tutorial04.txt:100]: `F` import is missing.

**Review by felixxm — APPROVED** [review]
@carltongibson Thanks :+1:

**Pull request merged**